SpatialGDK/Source/SpatialGDKEditor/Private/SnapshotGenerator/SpatialGDKEditorSnapshotGenerator.cpp

```diff
@@ -222,6 +222,38 @@ bool CreateSnapshotPartitionEntity(Worker_SnapshotOutputStream* OutputStream)
 	return Worker_SnapshotOutputStream_GetState(OutputStream).stream_state == WORKER_STREAM_STATE_GOOD;
 }
 
+bool CreateStrategyPartitionEntity(Worker_SnapshotOutputStream* OutputStream)
+{
+	Worker_Entity StrategyPartitionEntity;
+	StrategyPartitionEntity.entity_id = SpatialConstants::INITIAL_STRATEGY_PARTITION_ENTITY_ID;
+
+	TArray<FWorkerComponentData> Components;
+
+	AuthorityDelegationMap DelegationMap;
+	DelegationMap.Add(SpatialConstants::GDK_KNOWN_ENTITY_AUTH_COMPONENT_SET_ID, SpatialConstants::INITIAL_SNAPSHOT_PARTITION_ENTITY_ID);
```

Comment:
Setting the authority this way will assign it to the GSM system. So either we do that, and the GSM assigns these entities to the strategy worker itself as part of its startup sequence, or we assign this entity to itself, and make the StrategyWorker claim it as part of its own startup sequence.

Reply:
Discussed offline - putting it in the snapshot does seem better. The unreal workers should be taking their cues from the strategy worker rather than the other way around. However this isn't something that's essential to get right immediately as we still have old worker startup stuff.

Reply:
Done